dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -48,7 +48,7 @@ func ExtractIndexPattern(body []byte) ([]byte, error) {
 		if !ok {
 			return nil, fmt.Errorf("type key not found or not string")
 		}
-		if _type != "index-pattern" {
+		if _type != "index-pattern" || indexPattern == true {
```

Comment:
Why not write `|| indexPattern` instead of `|| indexPattern == true`?

Reply:
Yes, works too. IMHO I like to be explicit.

Reply:
I checked other parts of the code. Although, I find it weird, other parts of the code does include bool checking using `== true`, so I am ok with letting this go.